mne/io/utils.py

```diff
@@ -249,3 +249,18 @@ def _synthesize_stim_channel(events, n_samples):
     for onset, duration, trigger in events:
         stim_channel[onset:onset + duration] = trigger
     return stim_channel
+
+
+def _construct_bids_filename(base, ext, part_idx):
```

Comment:
I find `mne.io._construct_bids_filename` more natural than `mne.utils._construct_bids_filename`.

Reply:
it's a hidden helper function. it won't be exposed to the end user

Reply:
it would be placed as `mne.io.utils._construct_bids_filename` so it is within the scope of the io module. note, we have (at least) two different utils files.